src/main/java/org/ecocean/servlet/MergeIndividual.java

```diff
@@ -145,14 +150,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             for (int i=0; i<desiredIncrementalIdArr.size(); i++){
               if(!deprecatedIncrementIdsArr.get(i).equals("_")){
                 //there is a deprecated incremental ID that we need to rename and add to both individuals
-                mark1.addName("Merged " + projIdsArr.get(i),deprecatedIncrementIdsArr.get(i));
+                mark1.addName("Merged " + projNamesArr.get(i),deprecatedIncrementIdsArr.get(i));
```

Comment:
is there a check in here to make sure that `projNamesArr.size()` is **>=** `desiredIncrementalIdArr.size()` ?  seems like this `get(i)` will run risk of `i` being out of range if not.  i guess same goes for `deprecatedIncrementalIdsArr.get(i)` as well.  might be good to sanity-check that when the parameters are read in.